web/assignments_parser.go

```diff
@@ -15,6 +15,8 @@ import (
 const (
 	target     = "assignment.yml"
 	targetYaml = "assignment.yaml"
+	// default minimal score to be reached before assignment can be autoapproved
```

Comment:
Maybe this comment is unnecessary here. The const name is self-explanatory. But the doc comment could perhaps be rephrased and added elsewhere, e.g. for the `assignmentData` struct below.

Reply:
Done as well.